### DISCUSSION TITLE
How about having explicit separated fan on/off values?

### DISCUSSION THREAD

**Discussion by SaccoSVD** [Rem0o/FanControl.Releases#2131] [Ideas]
This for my 4090 Aorus Master.

I want a way to have fans turn on at 60c then turn off only when reaching 50c, or 40c, or whatever I decide the floor would be.

That way the fans will not turn on/off all the time. The idle temps on these cards are higher than previous gens.

Despite how many threads I've read or info about hysteresis, is not doing what I want in either software.

For instance, I have fans on at 60c and 10c hysteresis and the fans still turn on and off around 60c. Same on MSIAB. I tried all options in the "advanced hysteresis" without avail.

Seems like hysteresis alone is not doing the required job. If I don't understand and it is my fault, please can someone tell me how to achieve that? Seems a bit too complicated for something that should be simple.

Again, if we have an explicit way to tell the program "Turn on fans when the GPU temp reaches 60c and don't turn them off until temp reaches 50c" would be straightforward.

Thank you and sorry if this is a dumb question. I think I'm not the only one hoping for something like this.

EDIT: Disabling "Force fan speed update on each period" fixed it in MSIAB. So please disregard.

**Comment by Rem0o** [comment]
Try this as a starting point:
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/93ee9787-ccf2-4483-8fb5-f7a070e199d7)
